### PR TITLE
Some minor updates based on feedback from users

### PR DESCRIPTION
**Pull request by JosephDSchwartz**
Adding a couple of new properties:
1. `db-server-port`: this used to be set in the db-server-name, if needed, but the java connection string uses a different format (using a colon instead of a comma to separate the name and port), so we need to split these properties up.
2. `should-validate-migrations`: used to specify whether validation should be done when running migrations. The validation will check migrations that have been run for changes and throw an error if there have been any.

### CODE REVIEW

File: scripts/build-flyway.ps1 [hunk 1]
```diff
@@ -7,7 +7,9 @@ param(
     [string]$mockDbObjectNugetFeedUrl,
     [string]$nugetUser,
     [securestring]$nugetPassword,
-    [string]$dbServerName = "localhost"
+    [string]$dbServerName = "localhost",
+    [string]$dbServerPort = "1433",
+	[switch]$validateMigrations = $false
```
Comment (hpractv) [inline scripts/build-flyway.ps1:12]: I don't know how particular you want to be...but the indent here is different.

File: scripts/database-management/DMFlyway.ps1 [hunk 2]
```diff
@@ -69,6 +69,7 @@ function Invoke-DatabaseBuild {
         [switch]$seedData = $false,
         [switch]$runAllMigrations,
         [switch]$installMockDbObjects,
+		[switch]$validateMigrations = $false,
```
Comment (hpractv) [inline scripts/database-management/DMFlyway.ps1:72]: Same thing here with the indentation difference.

File: scripts/database-management/DMFlyway.ps1 [hunk 3]
```diff
@@ -433,6 +435,7 @@ function Invoke-Flyway {
     try
     {
         $outOfOrderValue = $enableOutOfOrder.ToString().ToLower()
+		$validateOnMigrateValue = $validateMigrations.ToString().ToLower()
```
Comment (hpractv) [inline scripts/database-management/DMFlyway.ps1:438]: Indentation...

File: scripts/database-management/DMFlyway.ps1 [hunk 4]
```diff
@@ -446,6 +449,7 @@ function Invoke-Flyway {
             "-baselineVersion=$baselineVersion"
             "-schemas=`"$managedSchemas`""
             "-outOfOrder=$outOfOrderValue"
+			"-validateOnMigrate=$validateOnMigrateValue"
```
Comment (hpractv) [inline scripts/database-management/DMFlyway.ps1:452]: Indentation... 😄

File: scripts/database-management/DMFlyway.ps1 [hunk 5]
```diff
@@ -129,11 +130,11 @@ function Invoke-DatabaseBuild {
             $versions = Get-Version -sqlFolder $baselineSnapshotFolder
             $baselineVersion = ($versions | Measure-Object -Maximum).Maximum
             Invoke-FlywayBaseline -hostName $hostName -port $port -dbName $dbName -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion
-            Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $baselineSnapshotFolder -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion
+            Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $baselineSnapshotFolder -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion -validateMigrations:$validateMigrations
```
Comment (hpractv) [inline scripts/database-management/DMFlyway.ps1:133]: This line is long and could use a manual wrap.

File: scripts/database-management/DMFlyway.ps1 [hunk 6]
```diff
@@ -129,11 +130,11 @@ function Invoke-DatabaseBuild {
...
         }
         Write-Status "Executing Flyway build and migrations";
         Write-Verbose "The command is: [Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $sqlFolder -MigrationHistoryTable ""MigrationHistory""]"
-        Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $sqlFolder -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion
+        Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $sqlFolder -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion -validateMigrations:$validateMigrations
```
Comment (hpractv) [inline scripts/database-management/DMFlyway.ps1:137]: This line is long and could use a manual wrap.

**Review by hpractv — DISMISSED** [review]
The code looks good.  I'm approving it but left some comments about indentation and wrapping.  You can decide if you want to change those before merging.

File: scripts/build-flyway.ps1 (hunk 1 above)
Comment (JosephDSchwartz) [reply scripts/build-flyway.ps1:12]: I think it has something to do with spaces vs. tabs. The scripts were originally written by people who were using tabs and now I'm modifying them using spaces. They look fine when you pull them down, but definitely strange here on github.

File: scripts/database-management/DMFlyway.ps1 (hunk 5 above)
Comment (JosephDSchwartz) [reply scripts/database-management/DMFlyway.ps1:133]: You're right, I'll do that for this and the other line below this.

**Review by hpractv — APPROVED** [review]

**Pull request merged**